src/software/thunderscope/thunderscope.py

```diff
@@ -6,17 +6,12 @@
 import platform
 import logging
 
-# PyQt5 doesn't play nicely with i3 and Ubuntu 18, PyQt6 is much more stable
 # Unfortunately, PyQt6 doesn't install on Ubuntu 18. Thankfully both
 # libraries are interchangeable, and  we just need to swap them in this
 # one spot, and pyqtgraph will pick up on it and store the library under
 # pyqtgraph.Qt. So from PyQt5 import x becomes from pyqtgraph.Qt import x
-if "18.04" in platform.version():
-    import PyQt5
-    from PyQt5.QtWebEngineWidgets import QWebEngineView
-else:
-    import PyQt6
-    from PyQt6.QtWebEngineWidgets import QWebEngineView
+import PyQt6
```

Comment:
I am worried that this change may affect other people's computer. Should I change it back to the old version?

Reply:
yes keep the old imports, but you can keep the old comment deleted

Reply:
sorry, that was bad communication from my part